src/NzbDrone.Core/Download/Pending/PendingReleaseService.cs

```diff
@@ -101,7 +105,7 @@ public List<Queue.Queue> GetPendingQueue()
 
                     var queue = new Queue.Queue
                                 {
-                                    Id = episode.Id ^ (pendingRelease.Id << 16),
+                                    Id = _uidProvider.Get(string.Format("pending-{0}-ep{1}", pendingRelease.Id , episode.Id)),
```

Comment:
I prefer the pending prefix, that way a future dev knows what it means, 'p' could be anything. It's just to avoid collisions between pending and CDH queue.